keyboards/work_louder/rgb_functions.c

```diff
@@ -14,10 +14,11 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
+#ifdef RGBLIGHT_ENABLE
+
 #include "rgblight.h"
 #include "rgb_matrix.h"
```

Comment:
```suggestion
```

Reply:
Applied suggestions, thanks for the review @zvecr.